cms/views.py

```diff
@@ -45,6 +47,47 @@ def visible(self, request):
         return Response(serializer.data)
 
 
+class LogoViewSet(viewsets.ModelViewSet):
+    """Publikácie(výsledky, brožúrky, časopisy, ...)"""
+    queryset = Logo.objects.all()
+    serializer_class = LogoSerializer
+    permission_classes = (PostPermission,)
+
+    def perform_create(self, serializer):
+        '''
+        Vola sa pri vytvarani objektu,
+        checkuju sa tu permissions, ci user vie vytvorit publication v danom evente
+        '''
+        event = serializer.validated_data['event']
+        if event.can_user_modify(self.request.user):
+            serializer.save()
+        else:
+            raise exceptions.PermissionDenied(
+                'Nedostatočné práva na vytvorenie tohoto objektu')
+
+    @action(methods=['get'], detail=True, url_path='download')
```

Comment:
Túto metódu netreba. Stačí ak v response príde link na súbor. Tento súbor sa totiž bude servovať staticky